### PR TITLE
Toast & rating added

### PR DESCRIPTION
**Pull request by msdeshmukh009**
**Toast**
- General toast
- Success toast
- Error toast
- Toast according to position: top-right toast, bottom-center toast, bottom-left toast.

**Rating**
- Read-only rating
- Live demo for rating

### CODE REVIEW

**Review by srishti-maurya — APPROVED** [review]
UI looks good

File: components/rating/rating.html [hunk 1]
```diff
@@ -0,0 +1,177 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (vidhi-thakur) [inline components/rating/rating.html:6]: you can also add one more meta tag here, check the image
image

File: components/rating/rating.html [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+            <a class="sidebar-links" href="/components/avatar/avatar.html">Avatar</a>
+          </li>
+          <li>
+            <a class="sidebar-links" href="/components/alert/alerts.html">Alerts</a>
+          </li>
+          <li>
+            <a class="sidebar-links" href="/components/badge/badges.html">Badges</a>
+          </li>
+          <li><a class="sidebar-links" href="/components/buttons/buttons.html">Button</a></li>
+          <li><a class="sidebar-links" href="/components/cards/cards.html">Cards</a></li>
```
Comment (vidhi-thakur) [inline components/rating/rating.html:46]: a minor change can be implemented here, format of <li> tags can be kept consistent. can use a code formatter here

**Review by vidhi-thakur — COMMENTED** [review]
added minor suggestions otherwise looks good to me

File: components/rating/rating.html (hunk 1 above)
Comment (msdeshmukh009) [reply components/rating/rating.html:6]: Ohh I didn't know this, will make this change.

File: components/rating/rating.html (hunk 2 above)
Comment (msdeshmukh009) [reply components/rating/rating.html:46]: Actually, this is because Prettier maybe I need to tweak some configuration there. Thanks for the review.

**Review by vidhi-thakur — APPROVED** [review]

**Review by Kavita539 — APPROVED** [review]

**Comment by Kavita539** [comment]
UI looks good! Good work with the structuring of the code.

**Pull request merged**